tests/test_unit/test_cli/test_behavior.py

```diff
@@ -279,16 +279,19 @@ def test_pull_workflow_trigger_with_environment_variables(
 
 
 def test_pull_group(
-    build_tmp_path: Path,
+    default_config_dev_yaml: str,
```

Comment:
This tests depended on which folder you executed it from. If you execute it from a folder without a `cdf.toml`, `config.dev.yaml` is not required and the tests passed - if not it failed. Adding it ensures it always passes.